modules/core/arrow-instances-core/src/main/kotlin/arrow/instances/char.kt

```diff
@@ -13,6 +11,10 @@ interface CharEqInstance : Eq<Char> {
   override fun Char.eqv(b: Char): Boolean = this == b
 }
 
+interface CharSemigroupInstance : Semigroup<Char> {
+  override fun Char.combine(b: Char): Char = this + b
```

Comment:
*squint* is that just...math?

Reply:
nope, and now that I think about it, this instance is impossible